doc/classes/JavaClassWrapper.xml

```diff
@@ -32,6 +32,8 @@
 			<param index="0" name="name" type="String" />
 			<description>
 				Wraps a class defined in Java, and returns it as a [JavaClass] [Object] type that Godot can interact with.
+				When wrapping inner (nested) classes, use [code]$[/code] instead of [code].[/code] to separate them.
+				For example, [code]JavaClassWrapper.wrap("android.view.WindowManager$LayoutParams")[/code] wraps [b]WindowManager.LayoutParams[/b] class.
```

Comment:
No need to be in two separate lines.
```suggestion
				When wrapping inner (nested) classes, use [code]$[/code] instead of [code].[/code] to separate them. For example, [code]JavaClassWrapper.wrap("android.view.WindowManager$LayoutParams")[/code] wraps the [b]WindowManager.LayoutParams[/b] class.
```